service-deviceregistry/Dockerfile

```diff
@@ -1,32 +1,25 @@
-# pull official base image
-FROM python:3.11.3-slim
+FROM python:3.11-alpine
 
-# create directory for the app user
-RUN mkdir -p /project
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
 
-# create the app user
-RUN addgroup --system app && adduser --system --group app
-
-ENV APP_HOME=/home/app
-WORKDIR $APP_HOME
-
-# copy project
-COPY . $APP_HOME
-
-# install dependencies
-COPY requirements.txt .
-RUN pip install --upgrade pip
-RUN pip install --no-cache-dir -r requirements.txt
+ENV DATABASE_NAME=postgres
+ENV DATABASE_USER=postgres
+ENV DATABASE_PASSWORD=postgres
+ENV DATABASE_HOST=db
+ENV DATABASE_PORT=5432
+ENV MEDIA_HOME=/media
+ENV DJANGO_SETTINGS_MODULE=deviceregistry.settings
 
-# chown all the files to the app user
-RUN chown -R app:app $APP_HOME
+RUN addgroup -S app && adduser -S app -G app
+USER app
+WORKDIR /home/app
```

Comment:
probably the tests fail because a test expect work dir to be /project

Reply:
Sounds likely. I'll merge and see that the tests work in PR #8.